### ISSUE TITLE
attention mask shape

### ISSUE THREAD

**Issue by graykode** [allenai/longformer#12]
https://github.com/allenai/longformer/blob/master/longformer/longformer.py#L88

In here, `attachment_mask` is known as the `[batch_size, seq_len]` shape. But I don't know why `squeeze` non-shape tensor.
Is it `unsqueeze` right that I guess?

**Comment by ibeltagy** [comment]
`attention_mask` here 
```
output = model(input_ids, attention_mask=attention_mask)
```
is of shape `[batch_size, seq_len]`, then the HF code here https://github.com/huggingface/transformers/blob/master/src/transformers/modeling_bert.py#L721 changes it to `[batch_size, 1, 1, seq_len]`. The line you mentioned returns it back to `[batch_size, seq_len]`

**Comment by graykode** [comment]
@ibeltagy Thanks for your reply! I didn't thought about that...
 I'll close this issue.
